### PR TITLE
Updated code to allow MacOS users to copy text in the textbox

### PR DESCRIPTION
**Pull request by ScraperMan2002**
I updated the .js code to allow MacOS users to copy text in the textbox. I bought a new Mac and realized that the copy function doesn't work at all.

### CODE REVIEW

File: hangulime.js
Comment (5hwb) [inline hangulime.js]: suggestion: Rename this to `isCmdPressed`

File: hangulime.js
Comment (5hwb) [inline hangulime.js]: nitpick: Add a `;` at the end of this line and other similar lines (e.g. lines 1340, 1370, 1372) to keep the code style consistent.

**Review by 5hwb — COMMENTED** [review]
Hey man, thanks for contributing and apologies for not getting back (been busy lately). Looks good, just got a few comments before I'm happy to approve

**Comment by ScraperMan2002** [comment]
OK I'm new to this sort of thing since Github changed a lot since you were gone. Now I have edited my fork, what do I do?

File: hangulime.js
Comment (5hwb) [inline hangulime.js]: issue: Thanks for catching this mistake. However, 2 keystrokes still point to the same jamo, here's what this line should be:

```suggestion
var jamo_med_yu_eo  = new Jamo(jamo_med_yu,  "bj", JAMO_MEDIAL, 'ᆏ', undefined);
```

File: hangulime.js
Comment (5hwb) [inline hangulime.js]: issue: As above, here's what this line should be:

```suggestion
var jamo_med_yu_yeo  = new Jamo(jamo_med_yu,  "bu", JAMO_MEDIAL, 'ᆑ', undefined);
```

File: hangulime.js
Comment (5hwb) [inline hangulime.js:1015]: issue: Had a look at this again and noted that this boolean isn't used anywhere, it should be removed

**Review by 5hwb — COMMENTED** [review]
> OK I'm new to this sort of thing since Github changed a lot since you were gone. Now I have edited my fork, what do I do?

Cool, you don't need to do anything after pushing your changes. I'll come and have another look at the PR and either approve it (after which you can merge in the changes into the main branch) or leave comments if I find any issues.

In this case, I've left some more comments, please have a look :speech_balloon:

**Comment by ScraperMan2002** [comment]
I fixed the changes

File: hangulime.js
Comment (ScraperMan2002) [reply hangulime.js:1015]: I thought that it was used in the code.

Comment (5hwb) [reply hangulime.js:1015]: The code is setting a value for `isCmdPressed`, but the `isCmdPressed` variable itself isn't being used anywhere else for decision making, not even for logging. It currently has no purpose being in the code

File: hangulime.js
Comment (5hwb) [inline hangulime.js]: issue(blocking): This changed line is wrong, please revert.

Comment (ScraperMan2002) [reply hangulime.js]: I don't think this is wrong. It has to do with how the word is displayed in MacOS natvie fonts. You see, in MacOS native fonts, U+1184 & U+1186 both look the exact same. However, if you type them out in the IME, you will find out that jamo_med_yo_ya and jamo_med_yo_yeo display the correct versions.

Comment (5hwb) [reply hangulime.js]: Then the MacOS native Hangul font must be defective. This IME is only concerned with inputting the correct and expected Hangul characters and should not account for any defects in Hangul fonts, as [U+1184 is clearly defined as a combination of YO ᅭ and YA ᅣ in Unicode](https://util.unicode.org/UnicodeJsps/character.jsp?a=1184) and not anything else.

In any case, your change now means that `jamo_med_yo_yeo` is defined twice (see lines 366 and 368) and `jamo_med_yo_ya` doesn't exist anymore, which results in an error that prevents the IME from working:

```
Uncaught ReferenceError: jamo_med_yo_ya is not defined
```

Comment (ScraperMan2002) [reply hangulime.js]: I see. I thought that error was fixed. I pushed my results now, should be fine.

**Review by 5hwb — APPROVED** [review]

**Comment by 5hwb** [comment]
I've approved the changes, you may merge in the PR anytime. Let me know if there's any issues, thanks for your contribution!

**Pull request merged**
